src/vdaf/mastic.rs

```diff
@@ -379,32 +391,500 @@ where
     }
 }
 
+/// Mastic prepare state.
+///
+/// State held by an aggregator between rounds of Mastic preparation. Includes intermediate
+/// state for Szk verification, the output shares currently being validated, and
+/// parameters of Mastic used for encoding.
+#[derive(Clone, Debug, Eq, PartialEq)]
+pub struct MasticPrepareState<F: FieldElement, const SEED_SIZE: usize> {
+    /// Includes output shares for eventual aggregation.
+    output_shares: MasticOutputShare<F>,
+    /// If SZK verification is being performed, we also store the relevant state for that operation.
+    szk_query_state: SzkQueryState<SEED_SIZE>,
+    verifier_len: Option<usize>,
+}
+
+/// Mastic prepare share.
+///
+/// Broadcast message from an aggregator between rounds of Mastic. Includes the
+/// hashed VIDPF proofs for every prefix in the aggregation parameter, and optionally
+/// the verification message for Szk.
+#[derive(Clone, Debug)]
+pub struct MasticPrepareShare<F: FieldElement, const SEED_SIZE: usize> {
+    /// Batched VIDPF proofs
+    vidpf_proofs: Seed<SEED_SIZE>,
+    /// If Szk verification of the root weight is needed, an SZK verification message.
```

Comment:
nit: I would normally leave a space between documented fields of a struct.
```suggestion
    vidpf_proofs: Seed<SEED_SIZE>, 
   
    /// If Szk verification of the root weight is needed, an SZK verification message.
```